test/v2/models/response/retrieveHistoricNonFhlUkPiePeriodSummary/PeriodExpensesSpec.scala

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v2.models.response.retrieveHistoricNonFhlUkPiePeriodSummary
+
+import play.api.libs.json.{ JsValue, Json }
+import support.UnitSpec
+import v2.models.utils.JsonErrorValidators
+
+class PeriodExpensesSpec extends UnitSpec with JsonErrorValidators {
+
+  private def decimal(value: String): Option[BigDecimal] = Option(BigDecimal(value))
+
+  val periodExpenses: PeriodExpenses =
+    PeriodExpenses(
+      decimal("5000.99"),
```

Comment:
Maybe use named parameters here and in other similar tests to make obvious comparison with JSON.

Reply:
Yes thats a good one to add too